### PR TITLE
cyaml: new recipe

### PR DESCRIPTION
**Pull request by Nat-B99**
Creation of a new recipe to use the cyaml library. 

See more : [link](https://github.com/tlsa/libcyaml)

### CODE REVIEW

File: meta-oe/recipes-support/cyaml/cyaml_1.0.bb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+SRC_URI = "git://github.com/tlsa/libcyaml.git;protocol=https;branch=main"
+
+S = "${WORKDIR}/git"
+
+inherit pkgconfig
+
+EXTRA_OEMAKE = "'PREFIX=""' 'DESTDIR=${D}' 'CFLAGS=${CFLAGS}' 'LIBDIR=${libdir}' 'INCLUDEDIR=${includedir}'"
+
+do_compile() {
+    oe_runmake VARIANT=release
```
Comment (kraj) [inline meta-oe/recipes-support/cyaml/cyaml_1.0.bb:21]: Pass `VARIANT=release` via `EXTRA_OEMAKE` as well.

File: meta-oe/recipes-support/cyaml/cyaml_1.0.bb
Comment (kraj) [inline meta-oe/recipes-support/cyaml/cyaml_1.0.bb:1]: latest version seems to be 1.4.1 see https://github.com/tlsa/libcyaml/releases/tag/v1.4.1 but the recipe says 1.0, lets fix it.

File: meta-oe/recipes-support/cyaml/cyaml_1.0.bb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+SUMMARY = "Cyaml library"
+DESCRIPTION = "LibCYAML is a C library for reading and writing structured YAML documents. It is written in ISO C11 and licensed under the ISC licence."
```
Comment (kraj) [inline meta-oe/recipes-support/cyaml/cyaml_1.0.bb:2]: I am not sure if license info and C std are needed in description. Instead add more about functionality if you wish

**Review by kraj — CHANGES_REQUESTED** [review]
Please address the feedback and thanks for your contribution !

**Review by kraj — COMMENTED** [review]
looks good. Please squash the changes into previous commit.

File: meta-oe/recipes-support/cyaml/cyaml_1.0.bb (hunk 1 above)
Comment (Nat-B99) [reply meta-oe/recipes-support/cyaml/cyaml_1.0.bb]: Okay done

File: meta-oe/recipes-support/cyaml/cyaml_1.0.bb
Comment (Nat-B99) [reply meta-oe/recipes-support/cyaml/cyaml_1.0.bb:1]: Okay done

File: meta-oe/recipes-support/cyaml/cyaml_1.0.bb (hunk 2 above)
Comment (Nat-B99) [reply meta-oe/recipes-support/cyaml/cyaml_1.0.bb]: Okay done

**Comment by kraj** [comment]
> looks good. Please squash the changes into previous commit.

I applied this with a21124a320
in future I will wait for submitters to address the feedback comments before accepting patches here.

**Comment by kraj** [comment]
applied with a21124a320

**Pull request closed without merging**